### PR TITLE
Feature/add tests

### PR DESCRIPTION
**Pull request by joshbDev**
Added Tests

### CODE REVIEW

**Comment by joshbDev** [comment]
this also addresses https://github.com/pricelinelabs/es-router/issues/7

**Comment by chrisrng** [comment]
Nice to see you around @thehindenburg

File: src/es-router.js [hunk 1]
```diff
@@ -58,31 +58,37 @@ class EsRouter {
...
-      window.onpopstate = this.eventChangeListener.bind(this);
+      this.returnWindow().onpopstate = this.eventChangeListener.bind(this);
     }
 
     //do an initial routing
     this.path(this.getPathFromUrl());
   }
 
+  //for testing purposes we can break into window
+  returnWindow() {
```
Comment (chrisrng) [inline src/es-router.js]: thoughts on keeping `window` intact but rather bootstrapping that instead?

Comment (joshbDev) [reply src/es-router.js]: good call, I mocked `GLOBAL.window` instead

**Pull request merged**